doc/experimental/phase1/phase1.md

```diff
@@ -17,6 +17,11 @@ Follow these [instructions](https://docs.oracle.com/en/operating-systems/olcne/2
 ocne cluster start --provider none --kubeconfig $KUBECONFIG --auto-start-ui false
 kubectl -n ocne-system rollout status deployment ocne-catalog
 ```
+
+## Perform an ETCD Backup
+Follow these [instructions](../../cluster-management/etcd-backup.md) to backup the ETCD database. 
+These backups should also be done periodically after the migration.
```

Comment:
Although this statement is true, I am not sure it is necessary.  There are many things that should be backed up periodically, regardless of a migration.

Reply:
Ok!